heimctl/retention/metrics.go

```diff
@@ -0,0 +1,15 @@
+package retention
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	roomHasExpiredMsg = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:      "room_has_expired",
+		Subsystem: "retention",
+		Help:      "Whether a room has expired messages, labelled by room name.",
```

Comment:
Had no idea there was a difference between American English and other forms. Changed- the presence command also uses labelled fyi. Found it via grep.